### PR TITLE
refactor: various component updates & add tab css

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

File: resources/views/status.blade.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    </div>
+@elseif ($type === 'updated')
+    <div class="flex items-center justify-center flex-shrink-0 border-2 rounded-full border-theme-warning-900 {{ $containerSize }}">
+        @svg('update', 'text-theme-warning-900 '.$size)
+    </div>
+@elseif ($type === 'locked')
+    <div class="flex items-center justify-center flex-shrink-0 border-2 rounded-full border-theme-secondary-700 {{ $containerSize }}">
+        @svg('lock', 'text-theme-secondary-700 '.$size)
+    </div>
+@endif --}}
```
Comment (ItsANameToo) [inline resources/views/status.blade.php:53]: what

File: resources/views/status-circle-shallow.blade.php [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
+        'xl'   => in_array($type, ['success', 'failed', 'error']) ? 'h-5 w-5' : 'w-6 h-6',
+        'base' => in_array($type, ['success', 'failed', 'error']) ? 'h-2 w-2' : 'h-3 w-3',
+    ][$size ?? 'base'];
+
+    $containerSize = [
+        'sm'   => 'w-4 h-4',
+        'md'   => 'w-8 h-8',
+        'lg'   => 'w-10 h-10',
+        'xl'   => 'w-12 h-12',
+        '2xl'   => 'w-14 h-14',
```
Comment (ItsANameToo) [inline resources/views/status-circle-shallow.blade.php:15]: spacing

File: resources/views/icon.blade.php [hunk 3]
```diff
@@ -5,6 +5,8 @@
         'sm'   => 'w-4 h-4 ',
         'md'   => 'w-6 h-6 ',
         'lg'   => 'w-8 h-8 ',
+        'xl'   => 'w-12 h-12',
+        '2xl'   => 'w-14 h-14',
```
Comment (ItsANameToo) [inline resources/views/icon.blade.php:9]: spacing

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: resources/views/status.blade.php (hunk 1 above)
Comment (alexbarnsley) [reply resources/views/status.blade.php:53]: that was intentional

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**